src/index.js

```diff
@@ -4,14 +4,17 @@ import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
 import { FiltersContextProvider } from "./contexts/filter-context";
+import { CartContextProvider } from "./contexts/cart-context";
```

Comment:
Suggestion: can use barrel export

so that you can directly import from `../contexts`